docker/docker-compose.yml

```diff
@@ -16,6 +17,8 @@ version: "3.8"
 services:
   postgres:
     image: "postgis/postgis:13-3.0"
+    ports:
+      - "5432:5432"
```

Comment:
This is the only part of the PR I'm not sure about, because it means if you already have a local Postgres server running (either in Docker or natively), then `docker-compose up` will fail to start this service because the host port is in use.

I do see the benefit of being able to connect to this database using local native tools on the host, e.g., IntelliJ's database browser.

If people are going to be using this as the main database instance in their dev environments, such that connecting to it with local tools is a regular desire, then it may be worth making this use a persistent volume for the database so the data doesn't get wiped on container restart.

Reply:
I added a persistent volume so the data doesn't get wiped. I also added a new `make` target `run-server` so that a user can _only_ run the server via Docker compose in case they're running Postgres natively or in Docker elsewhere

Reply:
Now that VirtioFS is supported in Mac and Docker (not experimental anymore), it technically isn't a bind mount anymore, the filesystem is accessed directly by the container. I think this is better because now there's no management of the volumes needed (no need to prune, the dir can just be deleted if you want), but it is _assuming_ that users are using this new-fangled tech in their Mac and Docker setup. 

All that being said, if you still want me to make this an OG volume, let me know!